test/conway_test.dart

```diff
@@ -1,5 +1,80 @@
 import 'package:test/test.dart';
 import 'package:conway/conway.dart';
+import 'package:matcher/matcher.dart';
+
+class _Endures extends Matcher {
+  const _Endures();
+
+  @override
+  Description describe(Description description) {
+    return description.add('endures');
+  }
+
+  @override
+  bool matches(dynamic pickle, Map matchState) {
+    WorldState world;
+    try {
+      world = WorldState.fromFixture(pickle);
+    } catch (e) {
+      matchState["parseError"] = e;
+      return false;
+    }
+    WorldState nextWorld = next(world);
+    String nextFixture = nextWorld.toFixture();
+    matchState["nextWorld"] = nextFixture;
```

Comment:
A bit odd that this is called "nextWorld" when it's a fixture string, not a world.